### DISCUSSION TITLE
Windows 11 support

### DISCUSSION THREAD

**Discussion by Alexandr-Konovalov** [microsoft/Detours#299] [Q&A] [answered]
In [README.md](https://github.com/microsoft/Detours#readme) I see that several versions of Windows is listed, but not Windows 11. Is it specific reasons for that? 

For me, `nmake test` is working on Windows 11 machine, but what is general status of Windows 11 support?

**Comment by sylveon** [comment, marked as answer]
It works fine on 11, Windows 11 is still NT 10.0
